ci/kokoro/ubuntu/build.sh

```diff
@@ -91,9 +91,6 @@ wget -q https://raw.githubusercontent.com/grpc/grpc/master/etc/roots.pem
 echo "    Getting cbt tool"
 export GOPATH="${KOKORO_ROOT}/golang"
 go get -u cloud.google.com/go/bigtable/cmd/cbt
-# TODO(#640) - disabled, the GCS library does not support credential files yet.
-# echo "    Getting python modules"
-# sudo python2 -m pip install httpbin
```

Comment:
Why is this pip module no longer required? Seems unrelated to enabling integration tests.

Reply:
Note that it was all commented out before.  I thought we would need it for the integration tests, turns out we won't, so I removed the TODO and the commented out code.